src/spec-lang/tc/typecheck.ts

```diff
@@ -1355,17 +1390,18 @@ export function tcMemberAccess(expr: SMemberAccess, ctx: STypingCtx, typeEnv: Ty
             }
         }
 
-        const type = BuiltinAddressMembers.get(expr.member);
+        // const type = BuiltinAddressMembers.get(expr.member);
```

Comment:
Commented code got left here :)

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/45/commits/3fda5277eb180bda7d656fc0d036f878c21290fe